modules/ROOT/pages/index.adoc

```diff
@@ -15,17 +15,17 @@ Silverblue is a variant of Fedora Workstation. It looks, feels and behaves like
 a regular desktop operating system, and the experience is similar to what you 
 find with using a standard Fedora Workstation.
 
-However, unlike other operating systems, Silverblue is immutable. This means 
-that every installation is identical to every other installation of the same 
-version. The operating system that is on disk is exactly the same from one 
-machine to the next, and it never changes as it is used.
-
-Silverblue's immutable design is intended to make it more stable, less prone to 
-bugs, and easier to test and develop. Finally, Silverblue's immutable design 
-also makes it an excellent platform for containerized applications as well as 
-container-based software development. In each case, applications (apps) and 
-containers are kept separate from the host system, improving stability and 
-reliability.
+However, unlike other operating systems, Silverblue has reprovisionable 
+infrastructure with anti-hysteresis properties. This means that every 
+installation is identical to every other installation of the same version. The 
+operating system that is on disk is exactly the same from one machine to the 
+next, and it never changes as it is used.
```

Comment:
Thanks so much for sending this PR!  We may want to take this to a hackmd.io doc or something where we can all collaboratively edit - I find that better than doing nitpicks via PR review.

Broadly speaking...I feel like we're going to confuse people if we *lead* with those terms.

Another bit here of course is "unlike other operating systems" depends on the PoV...it's more like Fedora CoreOS for example.

How about:

Defaults to transactional "image based" updates via rpm-ostree.  This gives you a reliable base system, with extensions (package layering) where you need it.  The focus is on containerized applications via a combination of `flatpak` for GUI applications, `podman` for local services, and `toolbox` as a general CLI environment.

Or something like that?

Reply:
I completely agree that using the terms, especially _anti-hysteresis_, can confuse or even scare people if that is the second sentence they ever read about silverblue.

This proposition was as simple as replacing "immutable" with "reprovisionable 
infrastructure with anti-hysteresis properties", which is of course not ideal.

I really like your proposition, as the terms used are much more convinient. Also mentioning that the focus is on containerized applications is imo good, but I think that the same point is made in the following paragraph - perhaps reformulate that one as well?

Anyway, thanks for looking into it :-)